<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>400</width>
      <height>230</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="dateLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="checkButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Check Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="dateTableWidget">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>60</y>
      <width>251</width>
      <height>331</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Разница</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
